src/components/header/HeaderContainer.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import HeaderView from './HeaderView';
+
+function HeaderComponent() {
```

Comment:
nit: function name doesn't match file name. Since this is consistently the case, is it intentional? I'm not very familiar with design patterns in React.

Reply:
It's not intentional. I initally had HeaderComponent.jsx and switched them to containers and missed some cases. Will update for sure.